### PR TITLE
[RISCV] Model integer min max instructions from Zbb execute in late-B ALU in SiFive7

### PR DESCRIPTION
**Pull request by michaelmaitland**
This is a stacked PR and I plan to commit changes separate.

commit a087d41fa69cbe017e9ced9f6e9965ad6d4168d0 (HEAD -> zbb-minmax, origin/zbb-minmax)
Author: Michael Maitland <michaeltmaitland@gmail.com>
Date:   Wed Mar 13 12:49:48 2024 -0700

    [RISCV] Model integer min max instructions from Zbb execute in late-B ALU

    We don't model the early vs late ALU so we just need to remove usage of
    SiFivePipeA for these instructions.

commit 75e2268ac6efe9edef4a8744999cd769ae184f25
Author: Michael Maitland <michaeltmaitland@gmail.com>
Date:   Wed Mar 13 12:48:33 2024 -0700

    [RISCV] Add sched classes for Zbb integer min max instructions

### CODE REVIEW

File: llvm/lib/Target/RISCV/RISCVSchedSiFive7.td [hunk 1]
```diff
@@ -1041,6 +1044,7 @@ def : SiFive7AnyToGPRBypass<ReadCTZ32>;
 def : ReadAdvance<ReadCPOP, 0>;
 def : ReadAdvance<ReadCPOP32, 0>;
 def : SiFive7AnyToGPRBypass<ReadORCB>;
+def : SiFive7AnyToGPRBypass<ReadIMinMax>;
```
Comment (topperc) [inline llvm/lib/Target/RISCV/RISCVSchedSiFive7.td:1047]: Need to add WriteIMinMax to the list in SiFive7AnyToGPRBypass

Comment (michaelmaitland) [reply llvm/lib/Target/RISCV/RISCVSchedSiFive7.td:1047]: Updated

**Review by topperc — APPROVED** [review]
LGTM

**Review by wangpc-pp — APPROVED** [review]
LGTM.

**Comment by michaelmaitland** [comment]
Committed in 620544192477cb8f4f1a1342e9593f7f15b5ecaf and 818e0272f5142986e8d82d1267fb6aa21cd168a0

**Pull request closed without merging**